### ISSUE TITLE
Listing issues from all sub projects

### ISSUE THREAD

**Issue by krthck** [dalyons/redmine-todos-scrum-plugin#39]
Currently, when creating a new entry, I get issues listed from current project. Is there a way to list all issues from subprojects also?
Even if it is code change, pls tell me the change, I will go ahead and do it.
Thanks.

**Comment by dalyons** [comment]
This would have a few consequences... If you allowed todos against subproject issues, then all todos from all subprojects would have to be listed in the todos page for the parent project. This could pollute the parent project/confuse people. 
Is this the behaviour you would be expecting?

**Comment by krthck** [comment]
Yes. I am using it for my project tracking. There are issues from multiple project for a single deliverable/sprint. It would be better if all are listed in the parent project.
Or is there any other way to achieve the same?
Thanks.
